lib/api/core/models/security/role.js

```diff
@@ -39,6 +52,15 @@ Role.prototype.isActionAllowed = function (requestObject, context) {
     return false;
   }
 
+  if (requestObject.persist &&
+    this.indexes._canCreate !== undefined &&
+    !this.indexes._canCreate &&
+    _.contains(['import', 'create', 'createCollection', 'putMapping', 'createOrUpdate'], requestObject.action) &&
```

Comment:
Can we find a better solution than list actions ? I'm not sure we'll remember to add a newly added collection here.  Maybe we can split the `adminController` into two files and have all these 5 methods in a specific controller and list available  method for this controller. Just a proposition, maybe we can discuss